prelims/handler.py

```diff
@@ -34,6 +35,9 @@ def load_posts(self):
         """
         posts = []
         for path in self.paths:
+            if path.name in self.ignore_files:
+                continue
```

Comment:
Can we test this in `prelims/tests/test_handler.py`?

Reply:
Added 1f9c892